app/assets/v2/js/cart.js

```diff
@@ -191,6 +191,15 @@ Vue.component('grants-cart', {
       }
       return result;
     },
+    fetchTokens() {
+      // removes duplicates from the tokens array of objects
+      let vm = this;
+      let uniq = {};
+
+      return vm.filterByChainId.filter(
+        obj => !uniq[obj['name']] && (uniq[obj['name']] = true)
+      );
```

Comment:
This is going to leave behind tokens which are not configured for polygon, will that present an issue? Instead of removing duplicates, could we filter by `chainId` AND `networkId`?